noir_stdlib/src/lib.nr

```diff
@@ -8,11 +8,19 @@ mod sha256;
 mod sha512;
 
 
-#[builtin(to_bits)]
-fn to_bits(_x : Field, _bit_size: u32) -> [u1; 256] {}
+ #[builtin(to_bits)]
+fn to_bits(_x : Field, _bit_size: u32) -> [u1] {}
+
+fn to_bytes(x : Field, byte_size: u32) -> [u8] {
```

Comment:
For reference, why does to_bits use a builtin while to_bytes uses to_radix ?

Reply:
This is because slices are not fully working yet, so to_radix outputs a u8 and then using to_radix would necessitate
a conversion to u1. The builtin can avoid it.